x-pack/plugins/ml/server/routes/schemas/results_service_schema.ts

```diff
@@ -52,3 +52,14 @@ export const partitionFieldValuesSchema = schema.object({
   earliestMs: schema.number(),
   latestMs: schema.number(),
 });
+
+export const getCategorizerStatsSchema = schema.nullable(
+  schema.object({
+    partitionByValue: schema.maybe(schema.string()),
+  })
+);
+
+export const getStoppedPartitionsSchema = schema.object({
```

Comment:
This should probably have a more specific name, like `getCategorizerStoppedPartitionsSchema`.

Reply:
Updated here https://github.com/elastic/kibana/pull/75061/commits/7d8c39b0fc8d740f237a4fe9f3c4ef6644c5a4ec